src/lib/othent-kms-client/operations/server-info.ts

```diff
@@ -0,0 +1,46 @@
+import { AxiosInstance } from "axios";
+import { CommonEncodedRequestData } from "./common.types";
+import { parseErrorResponse } from "../../utils/errors/error.utils";
+import { OthentAuth0Client } from "../../auth/auth0";
+import { Route } from "./common.constants";
+
+export interface ServerInfoOptions {
+  includeToken?: boolean;
+}
+
+export interface ServerInfo {
+  version: string;
+  buildDate: string;
+  hasToken: boolean;
+  hasTokenData: boolean;
+  isTokenValid: boolean;
+  isTokenUnused: boolean;
+}
+
+export async function serverInfo(
+  api: AxiosInstance,
+  auth0: OthentAuth0Client,
+  options?: ServerInfoOptions,
+): Promise<ServerInfo> {
+  const encodedData = options?.includeToken ? await auth0.encodeToken() : null;
+
+  let serverInfo: ServerInfo | null = null;
+
+  try {
+    const serverInfoResponse = encodedData
+      ? await api.post<ServerInfo>(Route.HOME, {
+          encodedData,
+        } satisfies CommonEncodedRequestData)
+      : await api.get<ServerInfo>(Route.HOME);
```

Comment:
Note the current server only supports the `GET` option here (the handler can take care of both `GET` and `POST` but I've just noticed I forgot to add the route). In any case, as this is an "internal" functionality for development, we can release the SDK change already and update the server later.